### PR TITLE
Andrew/backend update

### PR DESCRIPTION
**Pull request by Andrew-shanan**

### CODE REVIEW

**Comment by Andrew-shanan** [comment]
Please check the report, now you can work  on it's front end. Thank you

**Comment by Andrew-shanan** [comment]
is everything ok with the report?

**Comment by tatacsd** [comment]
What is the bug? i am running the application again now. @Andrew-shanan @luanrcard

File: backend/src/main/java/com/coffeelovers/pcu/PcuApplication.java [hunk 1]
```diff
@@ -48,55 +48,6 @@ ApplicationRunner init(
...
-	private void PopulateNoRalationalDataBase(ActorRepository actorRepository,
-			ProductRepository productRepository, 
-			SupplierRepository supplierRepository, 
-			VariableCostsRepository variableCostsRepository, 
-			InvoiceRepository invoiceRepository,
-			InvoiceDetailsRepository invoiceDetailsRepository,
-			RawMaterialRepository rowMaterialRepository) {
-
-		
-		//-1 Actor
```
Comment (tatacsd) [inline backend/src/main/java/com/coffeelovers/pcu/PcuApplication.java:70]: Why did you delete the actors? son?

File: backend/src/main/java/com/coffeelovers/pcu/PcuApplication.java [hunk 2]
```diff
@@ -48,55 +48,6 @@ ApplicationRunner init(
...
-		actorRepository.save(new Actor("Modlina","Women Fashion",559594, "www.shanan.com","shanan@gmail.com","ddd","canada" ));			
-		actorRepository.findAll().forEach(System.out::println);
-		
-		//2- product
-		productRepository.save(new Product("CCS2205","Skrit","ddd","222-220"));			
-		productRepository.findAll().forEach(System.out::println);
-		
-		
-		//3- suppliers
```
Comment (tatacsd) [inline backend/src/main/java/com/coffeelovers/pcu/PcuApplication.java:79]: and the suppliers?

**Review by tatacsd — APPROVED** [review]
